src/test/README.md

```diff
@@ -12,22 +12,20 @@ that runs all of the unit tests. The main source file for the test library is fo
 
 ### Compiling/running unit tests
 
-Unit tests will be automatically compiled if dependencies were met in `./configure`
+Unit tests will be automatically compiled if dependencies were met
+during the generation of the Bitcoin Core build system
 and tests weren't explicitly disabled.
 
-After configuring, they can be run with `make check`, which includes unit tests from
-subtrees, or `make && make -C src check-unit` for just the unit tests.
+Assuming the build directory is named `build`, the unit tests can be run
+with `ctests --test-dir build`, which includes unit tests from subtrees.
 
-To run the unit tests manually, launch `src/test/test_bitcoin`. To recompile
-after a test file was modified, run `make` and then run the test again. If you
-modify a non-test file, use `make -C src/test` to recompile only what's needed
-to run the unit tests.
+To run the unit tests manually, launch `build/src/test/test_bitcoin`.
 
 To add more unit tests, add `BOOST_AUTO_TEST_CASE` functions to the existing
 .cpp files in the `test/` directory or add new .cpp files that
 implement new `BOOST_AUTO_TEST_SUITE` sections.
 
-To run the GUI unit tests manually, launch `src/qt/test/test_bitcoin-qt`
+To run the GUI unit tests manually, launch `build/src/qt/test/test_bitcoin-qt`
```

Comment:
Nit: `Running individual tests` already covers this in the next section

Reply:
Isn't `test_bitcoin-qt` mentioned here only?

Reply:
Wrong line after quick rebase, I meant line 22.

Reply:
The "Running individual tests" section describes how to use `test_bitcoin` with different command line arguments. And it seems quite okay to me.

Anyway, this potential docs improvement can be done separately from the CMake migration, right?

Reply:
Of course, this comment was just a reformulation of your [original comment](https://github.com/hebasto/bitcoin/pull/322#discussion_r1713003805) - or at least what I understood from it, which was that we can delete this line since it's detailed later anwyay.